src/azul/indexer.py

```diff
@@ -160,6 +160,10 @@ def _create_indices(self):
                                      body=dict(settings=self.settings(index_name),
                                                mappings=dict(doc=self.mapping())))
 
+        # By default `max_compilations_per_minute` is set to 15, we must increase that threshold in order to execute
```

Comment:
As I wrote earlier, this method is used a lot and I do not want to do this every time a bundle is indexed. You mentioned in an earlier [comment](https://github.com/DataBiosphere/azul/pull/1442#discussion_r356196846) on this section

> I was unable to find resources that modify the cluster settings via terraform.

Please make sure that there really is no way this setting can be applied at cluster creation time. What you don't want is that someone else researches this for five minutes and comes up with a solution.

If there really is no way to do that, we should only limit the application of this setting to tests, and not in production code. Correct me if I am wrong, but I only recall us seeing the rate limit being hit during the tests.